conf/app.routes

```diff
@@ -1,3 +1,3 @@
 # microservice specific routes
 
-GET        /hello-world             uk.gov.hmrc.ecbforexrssstub.controllers.MicroserviceHelloWorldController.hello
+GET        /rss                  uk.gov.hmrc.ecbforexrssstub.controllers.RssFeedController.get
```

Comment:
I think this might need an extra bit on the url before /rss that shows it's part of our service. Just because I think when we did the /test-only stuff, we were fine locally but in staging it would literally just be the generic staging link with /rss which could be common across services? Just pointing it out, it may be fine! But just in case!

Reply:
The URL isn't really finalised yet - talked about it with Rob in that meeting on Friday. It can't match the URL of the external RSS feed as far as I'm aware, because it'll be running as a micro service on MTDP.

Reply:
okay, that's fine then, thanks!